Code/Engine/GameEngine/Animation/Implementation/PropertyAnimComponent.cpp

```diff
@@ -582,6 +587,10 @@ void ezPropertyAnimComponent::StartPlayback()
   {
     m_AnimationTime = m_AnimationRangeHigh;
   }
+  else
+  {
+    m_AnimationTime = m_AnimationRangeLow;
```

Comment:
Animation time is set here instead of in PlayAnimationRange since the range is now clamped to the correct duration.